### ISSUE TITLE
test rev G Hall test default configuration with two resistors removed board number 0 C7+C4

### ISSUE THREAD

**Issue by rusefillc** [rusefi/alphax-4chan#125]
Same physical unit as https://github.com/rusefi/alphax-4chan/issues/124

@ElDominio test hall C7+C4

**Comment by ElDominio** [comment]
C4 is VR-, did you mean c3?

**Comment by Qwerty-OFF** [comment]
С4 is IN_2STEP
![Screenshot_2023-04-22-21-26-44-954_com.google.android.apps.docs-edit.jpg](https://user-images.githubusercontent.com/4116383/233800657-28c4a55d-7e60-4a66-8902-34fa6564b48c.jpg)

**Comment by ElDominio** [comment]
Feed one of the signals through 2step?

**Comment by Qwerty-OFF** [comment]
Yes, use it as hall input

**Comment by ElDominio** [comment]
I can't choose C4 as input 
![image](https://user-images.githubusercontent.com/5685108/234981876-8d1f2b7d-aad0-4515-83e3-0f2cad7ebedf.png)

**Comment by ElDominio** [comment]
Hall test with revg
4B11 engine 36-2-1
working fine with revC

tune
[tunefor4b11revg.zip](https://github.com/rusefi/alphax-4chan/files/11348693/tunefor4b11revg.zip)

board0:
[revGhallv0.zip](https://github.com/rusefi/alphax-4chan/files/11348694/revGhallv0.zip)

board1:
[revGhallv1.zip](https://github.com/rusefi/alphax-4chan/files/11348695/revGhallv1.zip)

board2:
[revGhallv2.zip](https://github.com/rusefi/alphax-4chan/files/11348696/revGhallv2.zip)

**Comment by ElDominio** [comment]
tl;dr all good, did not try revving

**Comment by rusefillc** [comment]
@ElDominio why did not try revving?

**Comment by ElDominio** [comment]
On Hall i did not, but I can test

**Comment by ElDominio** [comment]
RevG works with Hall/VR
